README.md

```diff
@@ -481,8 +481,19 @@ terragrunt = {
 
 The `include` block tells Terragrunt to use the exact same Terragrunt configuration from the `terraform.tfvars` file
 specified via the `path` parameter. It behaves exactly as if you had copy/pasted the Terraform configuration from 
-the root `terraform.tfvars` file into `mysql/terraform.tfvars`, but this approach is much easier to maintain! Note
-that if you include any other settings in the `terragrunt` block of a child `.tfvars` file, it will override the
+the root `terraform.tfvars` file into `mysql/terraform.tfvars`, but this approach is much easier to maintain!
+
+The child `.tfvars` file's `terragrunt.terraform` settings will be merged into the parent file's `terragrunt.terraform`
+settings as follows:
+
+* If an `extra_arguments` block in the child has the same name as an `extra_arguments` block in the parent,
+  then the child's block will override the parents.
+  * Specifying an empty `extra_arguments` block in a child will effectively remove the parent's block.
```

Comment:
s/an empty `extra_arguments` block/an empty `extra_arguments` block of the same name/